vtrace/platforms/linux.py

```diff
@@ -413,6 +413,12 @@ def platformAttach(self, pid):
             raise Exception("PT_ATTACH failed!")
         self.setMeta("ExeName", self._findExe(pid))
 
+    def _LibraryLoadHack(self):
+        # drop special breakpoint at ld._dl_catch_exception
+        import vtrace.breakpoints as v_bp
```

Comment:
why can't this be a top level import?

Reply:
no reason, except it felt wrong to bring vtrace.breapoints into the platform when i wrote it.  i'll correct.

Reply:
done